### ISSUE TITLE
PM's not being marked as read

### ISSUE THREAD

**Issue by ocalhoun6** [knighty/fimfiction-issues#189]
This happened after I received 3 PM's from the same person at the same time earlier today.

Since then (when I opened all three in different tabs), I've had the notification ( http://s2.postimg.org/wna6ar8ft/screenshot_20.png ) for two unread PM's.

But when I open the list of PM's, all of them are marked as read. I've opened each and every PM there, didn't make the notification go away. I tried selecting them all and marking them as read -- still getting the notification.

Edit:
The top of the PM list still shows 2 unread as well, even though all the PM's below it are read.
http://s2.postimg.org/l39xwgos9/screenshot_20.png

**Comment by ocalhoun6** [comment]
I was finally able to get rid of the notification by using the 'mark all as read' button ( http://s2.postimg.org/5ydtyy0l5/screenshot_20.png ) on the pop-up under the notification itself.

**Comment by TrickQuestion** [comment]
I've posted a separate issue on this. I think it may explain why you're seeing this issue, however, so they might be the same bug.

**Comment by ocalhoun6** [comment]
No, not the same as you describe there -- in this case, I had opened every single incoming message. _Twice._ And they still didn't get marked as read. Even selecting them all and using 'actions --> mark as read' didn't work.
